encoder/vaapiencoder_vp8.cpp

```diff
@@ -47,6 +47,9 @@ class VaapiEncPictureVP8 : public VaapiEncPicture
 VaapiEncoderVP8::VaapiEncoderVP8():m_frameCount(0)
 {
     m_videoParamCommon.profile = VAProfileVP8Version0_3;
+    m_videoParamCommon.rcParams.minQP = 9;
```

Comment:
It's just a empirical value, because I think qp smaller than it will bring huge bitstream size increasing but litter quality improvement. It's ok to set to be 1, but never 0, beause QP is a numerator actually.